### PR TITLE
update schema OLH to v1.144.0

### PR DESCRIPTION
**Pull request by trend-dennis-tseng**

### CODE REVIEW

File: doc/Detections.yaml [hunk 1]
```diff
@@ -3570,6 +3590,7 @@
 - Name: logonUsers
   ProductCode:
   - ALL
+  - ddr
   Description_EN: The telemetry events match the Security Analytics Engine filter,
```
Comment (rebeccalee828) [inline doc/Detections.yaml:3594]: The telemetry events that match the Security Analytics Engine filter, and logonUsers stores the logonUsers value of the original events

File: doc/Detections.yaml [hunk 2]
```diff
@@ -2306,6 +2313,7 @@
   - sig
   - ams
   - szn
+  - ddr
   Description_EN: The host operating system name
```
Comment (rebeccalee828) [inline doc/Detections.yaml:2317]: The host OS name

File: doc/Cloud Activity Data.yaml [hunk 3]
```diff
@@ -162,7 +162,7 @@
   - sct
   Description_EN: Additional data about the event that was not part of the request
```
Comment (rebeccalee828) [inline doc/Cloud Activity Data.yaml:163]: The additional data about the event that was not part of the request

File: doc/Cloud Activity Data.yaml [hunk 4]
```diff
@@ -261,7 +261,7 @@
   - sct
   Description_EN: Identifies the service event, including what triggered the event
```
Comment (rebeccalee828) [inline doc/Cloud Activity Data.yaml:262]: The service event (including what triggered the event and the result)

**Review by rebeccalee828 — CHANGES_REQUESTED** [review]

File: doc/Cloud Activity Data.yaml (hunk 3 above)
Comment (trend-dennis-tseng) [reply doc/Cloud Activity Data.yaml:163]: We will update on this ticket: https://trendmicro.atlassian.net/browse/LUWA-19825

File: doc/Cloud Activity Data.yaml (hunk 4 above)
Comment (trend-dennis-tseng) [reply doc/Cloud Activity Data.yaml:262]: We will update on this ticket: https://trendmicro.atlassian.net/browse/LUWA-19825

File: doc/Detections.yaml (hunk 2 above)
Comment (trend-dennis-tseng) [reply doc/Detections.yaml:2317]: We will update on this ticket: https://trendmicro.atlassian.net/browse/LUWA-19825

File: doc/Detections.yaml (hunk 1 above)
Comment (trend-dennis-tseng) [reply doc/Detections.yaml:3594]: We will update on this ticket: https://trendmicro.atlassian.net/browse/LUWA-19825

**Review by rebeccalee828 — APPROVED** [review]

**Review by rebeccalee828 — APPROVED** [review]

**Pull request merged**